configs/apollodata.json

```diff
@@ -4,17 +4,23 @@
     "https://www.apollographql.com/docs/"
   ],
   "stop_urls": [
-    "index\\.html"
+    "#"
```

Comment:
I decided to use this based on a similar configuration I saw on the `meteor` configuration, but I'm not positive if this is correct or not (or even necessary!).

Reply:
`index\\.html` was there because you might have pages available under two deferent URLs. One with the trailing `index.html`, the other one without.
What are you aiming with this `#`?

Reply:
I copied this from https://github.com/algolia/docsearch-configs/blob/master/configs/meteor.json#L27, which is a _similar_ theme and configuration.  Happy to adjust this accordingly, as you think is appropriate.

With what you're saying though, having `index\\.html` seems to make sense for both this configuration and `meteor`?

Reply:
This "#" has no use anymore. Must be because of the way of rendering. [Removed here](https://github.com/algolia/docsearch-configs/commit/c4685752b67f86f16f48a9126766d30b752c1fe6).

Yep it makes sense if you can't avoid `index.html` to be available (200).